<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::CommandMappings</class>
 <widget class="QWidget" name="Core::Internal::CommandMappings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>451</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Command Mappings</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="Utils::FilterLineEdit" name="filterEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="commandList">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <column>
         <property name="text">
          <string>Command</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Label</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Target</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="defaultButton">
          <property name="toolTip">
           <string>Reset all to default</string>
          </property>
          <property name="text">
           <string>Reset All</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="importButton">
          <property name="text">
           <string>Import...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportButton">
          <property name="text">
           <string>Export...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="targetEditGroup">
     <property name="title">
      <string>Target Identifier</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="targetEditLabel">
          <property name="text">
           <string>Target:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Utils::FilterLineEdit" name="targetEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Reset to default</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::FilterLineEdit</class>
   <extends>Utils::FancyLineEdit</extends>
   <header location="global">utils/filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
